puppet/zulip_ops/manifests/firewall_allow.pp

```diff
@@ -0,0 +1,17 @@
+define zulip_ops::firewall_allow (
+  $port = '',
+  $proto = 'tcp',
+  $order = '50',
```

Comment:
How does this work when we have multiple `firewall_allow` calls?  (Is the resulting output order of those lines deterministic?)

Reply:
It's deterministic by name (which is unique) if they share an `order`: https://github.com/puppetlabs/puppetlabs-concat/blob/main/REFERENCE.md#order-1

Reply:
OK.  Maybe worth a comment.